Lesson1/Sviatoslav_Trenchuk_hm/functions/src/brackets.cpp

```diff
@@ -0,0 +1,51 @@
+#include "brackets.h"
+#include <vector>
+#include <stack>
+#include <iostream>
+
+bool areAllBracketsClosed(const std::string& inputString)
+{
+  std::vector<char> brackets {};
+  for (auto &value : inputString) {
+    brackets.push_back(value);
```

Comment:
**String is also a container (pseudo-container). You shouldn't copy it. Also, if you copy, use std::copy instead of the loop.**